imgurScraper.py

```diff
@@ -14,8 +14,7 @@
 # get the image paths
 imgPath = tree.xpath("//*[@class='zoom']/@href")
 # clears out the de-clutters the folder name
-folderName = folderName[0].text_content().rstrip(
-    'Imgur').rstrip('Album on').rstrip('-').strip()
+folderName = folderName[0].text_content().rstrip('Imgur').rstrip('Album on').rstrip('-').strip()
```

Comment:
Why did you make it in the same line? do an autopep8 to adhere to the python standard.